ui/src/app/components/runs/filters/multiple-status-filter/multiple-status-filter.component.ts

```diff
@@ -39,44 +39,41 @@ export class MultipleStatusFilterComponent implements ClrDatagridFilterInterface
     this.removeFiltersSubject.subscribe(_ => this.onRemoveFilter());
   }
 
-  toggleStatuses(statusModel) {
-    if (!statusModel.checked) {
-      statusModel.checked = true;
-      this.selectedValues = this.selectedValues.concat(statusModel.name);
-    } else {
-      statusModel.checked = false;
-      const index: number = this.selectedValues.indexOf(statusModel.name);
-      if (index >= 0) {
-        this.selectedValues = this.selectedValues.filter(status => status !== statusModel.name);
-      }
+  ngOnDestroy(): void {
+    this.removeFiltersSubject.unsubscribe();
+  }
+
+toggleStatuses(statusModel) {
+  if (this.selectedValues.indexOf(statusModel.name) < 0) {
+    this.selectedValues = this.selectedValues.concat(statusModel.name);
+  } else {
+    if (this.selectedValues.indexOf(statusModel.name) >= 0) {
```

Comment:
Do you need this if condition here? If you enter this else-block isn't `this.selectedValues.indexOf(statusModel.name)` always >= 0?

Reply:
Also, please auto-format this file